### PR TITLE
Intial commit to add environment descriptions to System>Support

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

File: userguide/system.rst [hunk 1]
```diff
@@ -2375,8 +2375,39 @@ To generate a support ticket, fill in the fields:
...
+      :class: longtable
+
+   +---------------------+----------------------------------------------------------+
+   | Environment         | Description                                              |
+   +=====================+==========================================================+
+   | Production          | The system is being used for production. Data loss       |
+   |                     | affects customers directly.                              |
+   |                     |                                                          |
+   +---------------------+----------------------------------------------------------+
+   | Staging             | The system is being prepared for                         |
```
Comment (Mrt134) [inline userguide/system.rst]:
```suggestion
   | Staging             | The system is being prepared for production.             |
```

File: userguide/system.rst [hunk 2]
```diff
@@ -2375,8 +2375,39 @@ To generate a support ticket, fill in the fields:
...
+   |                     | production.                                              |
```
Comment (Mrt134) [inline userguide/system.rst]:
```suggestion

```

File: userguide/system.rst [hunk 3]
```diff
@@ -2375,8 +2375,39 @@ To generate a support ticket, fill in the fields:
...
+   |                     |                                                          |
+   +---------------------+----------------------------------------------------------+
+   | Test                | The system is being used for for internal                |
```
Comment (Mrt134) [inline userguide/system.rst]:
```suggestion
   | Test                | The system is being used internally.                     |
```

File: userguide/system.rst [hunk 4]
```diff
@@ -2375,8 +2375,39 @@ To generate a support ticket, fill in the fields:
...
+   |                     | use. The system does not affect customers.               |
```
Comment (Mrt134) [inline userguide/system.rst]:
```suggestion
   |                     | The system does not affect customers.                    |
```

File: userguide/system.rst [hunk 5]
```diff
@@ -2375,8 +2375,39 @@ To generate a support ticket, fill in the fields:
...
+   |                     |                                                          |
+   +---------------------+----------------------------------------------------------+
+   | Prototyping         | The system is one of a kind. The use case is likely      |
```
Comment (Mrt134) [inline userguide/system.rst]:
```suggestion
   | Prototyping         | The system is unique. The use case is likely             |
```

File: userguide/system.rst [hunk 6]
```diff
@@ -2375,8 +2375,39 @@ To generate a support ticket, fill in the fields:
...
+   |                     | to be rare or never seen before.                         |
+   |                     |                                                          |
+   +---------------------+----------------------------------------------------------+
+   | Initial Deployment/ | The system is new and in the process of being configured |
```
Comment (Mrt134) [inline userguide/system.rst]:
```suggestion
   | Initial Deployment/ | The system is new and being configured for use.          |
```

File: userguide/system.rst [hunk 7]
```diff
@@ -2375,8 +2375,39 @@ To generate a support ticket, fill in the fields:
...
+   | Setup               | for use.                                                 |
```
Comment (Mrt134) [inline userguide/system.rst]:
```suggestion
   | Setup               |                                                          |
```

File: userguide/system.rst [hunk 8]
```diff
@@ -2375,8 +2375,38 @@ To generate a support ticket, fill in the fields:
...
+.. _environment options:
+
+.. table:: Environment Options
+      :class: longtable
+
+   +---------------------+----------------------------------------------------------+
+   | Environment         | Description                                              |
+   +=====================+==========================================================+
+   | Production          | The system is being used for production. Data loss       |
+   |                     | affects customers directly.                              |
```
Comment (yocalebo) [inline userguide/system.rst]: The sentence for the "Production" environment reads a little harshly and implies there is data loss when there is a production outage. My recommendation would be to say something like `The system is functional and being used in a production environment.`

File: userguide/system.rst [hunk 9]
```diff
@@ -2375,8 +2375,38 @@ To generate a support ticket, fill in the fields:
...
+   |                     |                                                          |
+   +---------------------+----------------------------------------------------------+
+   | Staging             | The system is being prepared for production.             |
+   |                     |                                                          |
+   +---------------------+----------------------------------------------------------+
+   | Test                | The system is being used internally.                     |
+   |                     | The system does not affect customers.                    |
+   |                     |                                                          |
```
Comment (yocalebo) [inline userguide/system.rst]: The sentence for the "Test" section is ambiguous and confusing, imo. My recommendation is to saying something like `The system is being used internally for testing purposes.` Remove the sentence `The system does not affect customers.`

File: userguide/system.rst [hunk 10]
```diff
@@ -2375,8 +2375,38 @@ To generate a support ticket, fill in the fields:
...
+   +---------------------+----------------------------------------------------------+
+   | Prototyping         | The system is unique. The use case is likely             |
+   |                     | to be rare or never seen before.                         |
+   |                     |                                                          |
```
Comment (yocalebo) [inline userguide/system.rst]: The definition for "Prototyping" in this context makes it seem like the system is "unique" or is an "alpha" model of the machine. This needs to be discussed further as I don't believe this section makes much sense.

This section is probably referring to "Proof of Concept". However, this section should be removed until the original intent can be understood.

File: userguide/system.rst [hunk 11]
```diff
@@ -2375,8 +2375,38 @@ To generate a support ticket, fill in the fields:
...
+   +---------------------+----------------------------------------------------------+
+   | Initial Deployment/ | The system is new and being configured for use.          |
+   | Setup               |                                                          |
+   |                     |                                                          |
```
Comment (yocalebo) [inline userguide/system.rst]: My recommendation for the sentence would be something like `The system is new and is being configured for the production environment.`

File: userguide/system.rst (hunk 10 above)
Comment (a-stjohn) [reply userguide/system.rst]: Those are my thoughts as well. I also thought that the "Prototyping" environment would be very similar to the "Testing" environment. Thus, if support doesn't have a concrete explanation of what the environment means and it's similar to "Testing", I think we can remove the option from the UI entirely.

File: userguide/system.rst [hunk 12]
```diff
@@ -2375,8 +2375,38 @@ To generate a support ticket, fill in the fields:
...
+
+.. _environment options:
+
+.. table:: Environment Options
+      :class: longtable
+
+   +---------------------+----------------------------------------------------------+
+   | Environment         | Description                                              |
+   +=====================+==========================================================+
+   | Production          | The system is functional and being used in a production  |
```
Comment (wblock) [inline userguide/system.rst]: This is a production system in daily use.

File: userguide/system.rst [hunk 13]
```diff
@@ -2375,8 +2375,38 @@ To generate a support ticket, fill in the fields:
...
+   |                     | environment.                                             |
+   |                     |                                                          |
+   +---------------------+----------------------------------------------------------+
+   | Staging             | The system is being prepared for production.             |
+   |                     |                                                          |
+   +---------------------+----------------------------------------------------------+
+   | Test                | The system is being used internally for testing          |
```
Comment (wblock) [inline userguide/system.rst]: This system is only being used for testing purposes.

File: userguide/system.rst (hunk 10 above)
Comment (wblock) [reply userguide/system.rst]: Agreed.  The overlap between Testing and Prototyping is only going to confuse the user unless we add concrete descriptions of what each means.

File: userguide/system.rst [hunk 14]
```diff
@@ -2375,8 +2375,38 @@ To generate a support ticket, fill in the fields:
...
+   |                     | purposes.                                                |
+   |                     |                                                          |
+   +---------------------+----------------------------------------------------------+
+   | Prototyping         | The system is unique. It is likely to be a proof of      |
+   |                     | concept.                                                 |
+   |                     |                                                          |
+   +---------------------+----------------------------------------------------------+
+   | Initial Deployment/ | The system is new and being configured for the           |
```
Comment (wblock) [inline userguide/system.rst]: This is a new system being prepared for deployment into production.

**Pull request merged**